.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,7 +31,7 @@ Of course, you can opt not to do this and be listed as an "Anonymous contributor
 
 <!-- Please check ALL the boxes: -->
 
-- [ ] All writings are my own.
+- [ ] All writing is my own.
```

Comment:
Thank you for taking the time to contribute to Codecademy docs. However I think this is correct.

Reply:
OK, thanks! :)